metrics/README.md

```diff
@@ -0,0 +1,62 @@
+# Clear Containers metrics tests
+
+This is a collection of scripts designed to measure performance and responsiveness
+of the different components which form Clear Containers.
+
+## Metrics tests - content
+
+The tests are organized in the following sections:
+
+| Section   | Short description
+| ----------| --------------------------------------------------------------------------------
+| density   | Measure the amount of memory used by N containers under different configurations.
+| network   | Measure a set of network essentials such as: bandwidth, jitter, latency etc.
+| storage   | Measure storage bandwith using different configurations.
+| time      | Measure the latency that a container[s] take[s] to do a defined workload.
+
+## HOWTO execute metrics scripts
+
+The metrics directory includes a helper script `run_all_metrics.sh`. It allows to execute all metrics
+tests or only a specific section.
+
+NOTE: Some metrics tests require root access to run correctly.
+
+## Options
+
+if no options are specified, `run_all_metrics.sh` will execute all tests by default.
+
+### Only run the metrics tests from time section
```

Comment:
Run the metrics tests from the time section only
Also, Please leave a blank line after all headings. They are missing from this point on.